source/ada/lsp-ada_handlers.ads

```diff
@@ -309,14 +329,15 @@ private
       File_Monitor    : LSP.File_Monitors.File_Monitor_Access;
       --  Filesystem monitoring
 
+      Dynamic_Registration_Ids : LSP.Ada_Handlers.Dynamic_Registration_Ids;
```

Comment:
A short comment is appreciated here. Also, I'm not sure if it worse a dedicated field at all, because it's used only once.

Reply:
Agree.
I'll remove this and hard code the ids when we register the capabilities.
If we ever need to deregister capabilities, then we come back to this approach to avoid duplicated hard coded strings.

Reply:
Done